tools/bin/ecflow/processes/main/post_processing/swe_plot.py

```diff
@@ -0,0 +1,207 @@
+        
+	#10mm threshold
+        if (value < 10):
+	    combine = (lat, lon)
+            d.append(combine)
+        else:    
+        
+            try:
+                #interpolation
+                f = interp1d(x,q)
+                f(value)
+                percentile = f(value)
+                combine = (lat, lon, float(percentile))
+                d.append(combine)
+            #if interpolation fails then a value is assigned based on whether it is higher or lower than the range
+            except ValueError:
+                if (value > max(x)):
+                    percentile = (max_q)
+                    combine = (lat, lon, percentile)
+                    d.append(combine)
+                else:
+                    percentile = (min_q)
+                    combine = (lat, lon, percentile)
+                    d.append(combine)
+    except IOError: 
+	percentile = value
+        combine = (lat, lon)
+        d.append(combine)
+
+#Dictionary to DataFrame to Dataset
+df = pd.DataFrame(d, columns=["Latitude", "Longitude", "Percentile"])
+a = df['Percentile'].values
+new = a.reshape(195, 245)
+
+dsx = xr.Dataset({'percentile': (['lat', 'lon'], new)}, coords={'lon': (['lon'], un_lon), 'lat': (['lat'], un_lat)})
+
+#plotting
+plt.figure(figsize=(8,8))
+
+ax = plt.axes(projection=ccrs.Mercator(central_longitude=-120, min_latitude=40.7, max_latitude=49.3, globe=None)) 
+
+ax.set_extent([-109.5, -125.01, 40, 54.01],ccrs.Geodetic())
+
+gl=ax.gridlines(draw_labels=True, xlocs = [-100, -110, -115, -120, -125], ylocs = [40, 42, 45, 48, 50])
+gl.xlabels_top = False
+gl.ylabels_right = False
+
+gl.xformatter = LONGITUDE_FORMATTER
+gl.yformatter = LATITUDE_FORMATTER
+
+states_provinces = cfeature.NaturalEarthFeature(
+        category='cultural',
+        name='admin_1_states_provinces_lines',
+        scale='50m',
+        facecolor='none')
+
+country_borders = cfeature.NaturalEarthFeature(
+        category='cultural',
+        name='admin_0_boundary_lines_land',
+        scale='50m',
+        facecolor='none')
+
+land = cfeature.NaturalEarthFeature(
+    category='physical',
+    name='land',
+    scale='50m',
+    facecolor='gray')
+
+ocean = cfeature.NaturalEarthFeature(
+    category='physical',
+    name='ocean',
+    scale='50m',
+    facecolor='blue')
+
+ax.add_feature(states_provinces, edgecolor='black', zorder = 2)
+ax.add_feature(country_borders, edgecolor='black', zorder = 2)
+ax.add_feature(land,facecolor='gray', zorder = 1)
+ax.add_feature(ocean,facecolor='lightblue', zorder = 1)
+
+
+cmap = matplotlib.colors.ListedColormap(['darkred', 'red', 'tomato', 'lightsalmon', 'mistyrose', 
+                                         'yellow', 'lightsteelblue', 'cornflowerblue', 'royalblue', 'blue', 'navy'])
+cmap.set_bad('lightgrey')
+img=dsx['percentile'].plot(ax = ax, vmin = 0, vmax = 10, levels=[0, 0.01, 0.05, 0.1, 0.2, 0.35, 0.65, 0.8, 0.9, 0.95, 0.99, 1.0], add_colorbar=False, cmap=cmap, transform = ccrs.PlateCarree(), zorder=2)
```

Comment:
space before and after first "=", and then not around ax=ax, vmin=0, vmax=10, etc.